clicache/integration-test/ThinClientDeltaTestN.cs

```diff
@@ -832,50 +832,55 @@ void runExpirationWithDelta()
 
     //#region Tests
 
-    [Test]
+    //[Test]
```

Comment:
I think I rather have tests that aren't working properly isolated to other file and marked in `CMakeLists.txt` as `omitted` with this commented in rather than commented out (assuming this isn't a mistake and is suppose to be in).  Same for the other `//[Test]`

Reply:
Good plan.